src/madhuraMJun20/scannerClass/PrimeNumber.java

```diff
@@ -0,0 +1,62 @@
+package madhuraMJun20.scannerClass;
+/*Program 1: Create a class to satisfy below requirements. 
+a) Given number is prime or not. 
+b) Print total count of prime numbers between 2->100.
+c) Find sum of all prime numbers between 2->100
+d) Find avg of prime numbers between 2->100*/
+
+public class PrimeNumber {
+	int count;
+	int sum;
+
+	void isPrimeNumber(int num) {
+		boolean flag = false;
+		for (int i = 2; i <= num / 2; i++) {
+			if (num % i == 0) {
+				flag = true;
+				break;
+			}
+		}
+		if (flag == false) {
+			System.out.println(num + " is a prime number");
+			count++;
+		} else
+			System.out.println(num + " is not a prime number");
+	}
+
+	void countPrime(int num1, int num2) {
+		for (int i = num1; i <= num2; i++) {
+			isPrimeNumber(i);
+		}
+		System.out.println("Total Prime numbers between 2->100 : " + count);
+	}
+
+	void sumOfPrimeNum(int num1, int num2) {
+		boolean flag;
+		for (int i = num1; i <= num2; i++) {
+			flag = true;
+			for (int j = num1; j <= i / 2; j++) {
```

Comment:
Here you should use isPrimeNumber() method.

Reply:
Updated the code with changes.